Public/Src/Cache/ContentStore/Distributed/NuCache/EventStreaming/EventHubContentLocationEventStore.cs

```diff
@@ -43,6 +46,8 @@ public sealed class EventHubContentLocationEventStore : ContentLocationEventStor
 
         private readonly ActionBlock<ProcessEventsInput>[] _eventProcessingBlocks;
 
+        private readonly Microsoft.Practices.TransientFaultHandling.RetryPolicy _extraEventHubClientRetryPolicy;

```

Comment:
Why both using definition and full name here?

Reply:
Removed. Had a conflict but the conflict was resolved by `using RetryPolicy=...` at the top of the file.